src/nise/metadata/enums_addon.py

```diff
@@ -0,0 +1,139 @@
+# These dictionaries are applied to the generated enums dictionary at build time
```

Comment:
What's up with this? Extraction didn't work so you had to type all this by hand?
Please:

* Make sure the names match the C NISE API
* Add comment to this file on why it's all written here and not in enums.py
* File internal bug report to NISE to fill out the proper source of metadata extraction.

Alternatively, we could source the enums as extracted in-house and export them just the same as we export the other ones. This one would just be hand-coded.

Reply:
Enums are extracted from attributes. 
There are no attributes in NI Switch Executive, therefore there are no extracted enums. 

There are no enum names to match to the C API, there were hand typed based on the LabVIEW API.

For other drivers, enums_addon.py was hand typed as well and was used for enums that are part of functions. The names for other drivers were also referencing the LabVIEW API. This is similar to other drivers.

The other drivers did not add info on why additional enums were needed. Should they have?

Reply:
> The other drivers did not add info on why additional enums were needed. Should they have?

I didn't remember this flaw in the extracted metadata... so I'd say that yes, it would probably be a helpful comment. Something along the lines of: "Extracted metadata is missing enums used only in functions. Those are added here manually."